frontend/exporter/src/traits.rs

```diff
@@ -253,8 +253,17 @@ impl<'tcx> IntoImplExpr<'tcx> for rustc_middle::ty::PolyTraitRef<'tcx> {
                         })
                         .map(|path| (predicate, path))
                 }) else {
-                    return ImplExprAtom::Todo(format!("implsource::param \n\n{:#?}", self))
-                        .with_args(impl_exprs(s, &nested));
+                    return
+                        if predicates.is_empty() {
+                            ImplExprAtom::LocalBound {
+                                clause_id: rand::random(), // TODO
+                                path: Vec::<search_clause::PathChunk>::new().sinto(s)
+                            }
+                        }
+                        else {
+                            ImplExprAtom::Todo(format!("implsource::param \n\n{:#?}", self))
+                        }
+                        .with_args(impl_exprs(s, &nested))
```

Comment:
That removes the error, but a `LocalBound` is not what we need (and the `random` will not help :smiley:)